include/bluetooth/audio/aics.h

```diff
@@ -0,0 +1,450 @@
+#define BT_AICS_INPUT_TYPE_DIGITAL                 0x03
+#define BT_AICS_INPUT_TYPE_RADIO                   0x04
+#define BT_AICS_INPUT_TYPE_PHYS_MEDIA              0x05
+#define BT_AICS_INPUT_TYPE_NETWORK                 0x06
+#define BT_AICS_INPUT_TYPE_OTHER                   0x255
+
+/** Audio Input Control Service Error codes */
+#define BT_AICS_ERR_INVALID_COUNTER                0x80
+#define BT_AICS_ERR_OP_NOT_SUPPORTED               0x81
+#define BT_AICS_ERR_MUTE_DISABLED                  0x82
+#define BT_AICS_ERR_OUT_OF_RANGE                   0x83
+#define BT_AICS_ERR_GAIN_MODE_NO_SUPPORT           0x84
+
+/** @brief Opaque Audio Input Control Service instance. */
+struct bt_aics;
+
+/** @brief Structure for initializing a Audio Input Control Service instance. */
+struct bt_aics_init {
+	/** Initial audio input gain (-128 to 127) */
+	int8_t gain;
+
+	/** Initial audio input mute state */
+	uint8_t mute;
+
+	/** Initial audio input mode */
+	uint8_t mode;
+
+	/** Initial audio input gain units (N * 0.1 dB) */
+	uint8_t units;
+
+	/** Initial audio input minimum gain */
+	int8_t min_gain;
+
+	/** Initial audio input maximum gain */
+	int8_t max_gain;
+
+	/** @brief Initial audio input type */
+	int8_t input_type;
+
+	/** Initial audio input state (enabled/disabled) */
+	bool input_state;
+
+	/** Boolean to set whether the description is writable by clients */
+	bool desc_writable;
+
+	/** Initial audio input description */
+	char *input_desc;
+};
+
+/** @brief Structure for discovering a Audio Input Control Service instance. */
+struct bt_aics_discover_param {
+	/** @brief The start handle of the discovering.
+	 *
+	 * Typically the @p start_handle of a @ref bt_gatt_include.
+	 */
+	uint16_t start_handle;
+	/** @brief The end handle of the discovering.
+	 *
+	 * Typically the @p end_handle of a @ref bt_gatt_include.
+	 */
+	uint16_t end_handle;
+};
+
+/**
+ * @brief Get a free instance of Audio Input Control Service from the pool.
+ *
+ * @return Audio Input Control Service instance in case of success or NULL in case of error.
+ */
+struct bt_aics *bt_aics_free_instance_get(void);
+
+/**
+ * @brief Get the service declaration attribute.
+ *
+ * The first service attribute returned can be included in any other GATT service.
+ *
+ * @param aics Audio Input Control Service instance.
+ *
+ * @return Pointer to the attributes of the service.
+ */
+void *bt_aics_svc_decl_get(struct bt_aics *aics);
+
+/**
+ * @brief Initialize the Audio Input Control Service instance.
+ *
+ * @param aics      Audio Input Control Service instance.
+ * @param init      Audio Input Control Service initialization structure.
+ *
+ * @return 0 if success, ERRNO on failure.
+ */
+int bt_aics_init(struct bt_aics *aics, struct bt_aics_init *init);
+
+/**
+ * @brief Callback function for writes.
+ *
+ * @param conn        Connection to peer device, or NULL if local server write.
+ * @param inst        The service pointer (as there may be multiple).
```

Comment:
"The instance pointer" (as for VOCS), or "Pointer to the service instance"?
(Multiple places.)